incubator/jenkins-operator/CRD-protected/crds.yaml

```diff
@@ -0,0 +1,177 @@
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+  name: jenkinsinstances.jenkins.jenkinsoperator.maratoid.github.com
+spec:
+  group: jenkins.jenkinsoperator.maratoid.github.com
+  names:
+    kind: JenkinsInstance
+    plural: jenkinsinstances
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            adminemail:
+              type: string
+            adminsecret:
+              type: string
+            annotations:
+              type: object
+            env:
+              type: object
+            executors:
+              format: int32
+              type: integer
+            image:
+              pattern: .+:.+
+              type: string
+            ingress:
+              properties:
+                annotations:
+                  type: object
+                path:
+                  type: string
+                service:
+                  type: string
+                tlssecret:
+                  type: string
+              type: object
+            location:
+              type: string
+            networkpolicy:
+              type: boolean
+            pluginconfig:
+              properties:
+                config:
+                  type: string
+                configsecret:
+                  type: string
+              type: object
+            plugins:
+              items:
+                properties:
+                  id:
+                    type: string
+                  version:
+                    type: string
+                type: object
+              type: array
+            service:
+              properties:
+                annotations:
+                  type: object
+                name:
+                  type: string
+                nodeport:
+                  format: int32
+                  type: integer
+                servicetype:
+                  type: string
+              type: object
+            serviceaccount:
+              type: string
+            storage:
+              properties:
+                jobspvc:
+                  type: string
+                jobspvcspec:
+                  type: object
+              type: object
+          type: object
+        status:
+          properties:
+            adminsecret:
+              type: string
+            phase:
+              type: string
+          required:
+          - phase
+          type: object
+      type: object
+  version: v1alpha1
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: null
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    helm.sh/resource-policy: keep
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
```

Comment:
Please add standard labels to resources:
https://docs.helm.sh/chart_best_practices/#standard-labels